pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -80,8 +80,8 @@ const (
 	maxTargetPoolCreateInstances = 200
 
 	// HTTP Load Balancer parameters
-	// Configure 2 second period for external health checks.
-	gceHcCheckIntervalSeconds = int64(2)
+	// Configure 5 second period for external health checks.
+	gceHcCheckIntervalSeconds = int64(5)
 	gceHcTimeoutSeconds       = int64(1)
 	// Start sending requests as soon as a pod is found on the node.
 	gceHcHealthyThreshold = int64(1)
```

Comment:
We should be reducing the gceHcUnhealthyThreshold to 3, otherwise it will be very unresponsive.

Reply:
Done.